src/main/java/org/breedinginsight/brapi/v2/dao/BrAPIGermplasmDAO.java

```diff
@@ -157,21 +157,29 @@ private Map<String,BrAPIGermplasm> processGermplasmForDisplay(List<BrAPIGermplas
 
                 String newPedigreeString = "";
                 String namePedigreeString = "";
+                String uuidPedigreeString = "";
                 List<String> parents = Arrays.asList(germplasm.getPedigree().split("/"));
                 if (parents.size() >= 1) {
                     if (programGermplasmByFullName.containsKey(parents.get(0))) {
                         newPedigreeString = programGermplasmByFullName.get(parents.get(0)).getAccessionNumber();
                         namePedigreeString = programGermplasmByFullName.get(parents.get(0)).getDefaultDisplayName();
+                        uuidPedigreeString = programGermplasmByFullName.get(parents.get(0)).getExternalReferences().
+                                stream().filter(ref -> ref.getReferenceSource().equals(referenceSource)).
```

Comment:
Looks good! Did you mean to have the `.` on a separate line from the `map` and `stream`? I usually have seen it on the same line. If it's a style choice, feel free to keep it.